release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -409,6 +409,29 @@ Be aware that CIDR ranges for networks are not adjustable after cluster installa
 
 If you are using the OVN-Kubernetes cluster network provider, you can now enable IPsec encryption after cluster installation. For more information about how to enable IPsec, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
 
+[id="ocp-4-11-new-metallb-CRDs-logging-control"]
+==== Support for new MetalLB custom resource defintions (CRDs) and control over logging verbosity
+
+MetalLB CRDs have been updated with some new CRDs to support more complex configuration. 
+
+The following new CRDs have been added: 
+
+* IPAddressPools
+* L2Advertisement
+* BgpAdvertisement
+* Community
+
+With these enhancements, users can use the Operator for more complex configurations, for example isolating nodes, or segmenting the network. In addition enhancements made to the FRRouting (FRR) logging component allows you to control the verbosity of the logs generated.
+
+[NOTE]
+====
+CRDs documented in 4.10 and the existing ways to configure MetalLB as described in link:https://docs.openshift.com/container-platform/4.10/networking/metallb/about-metallb.html[About MetalLB and the MetalLB Operator] are still supported but deprecated. The `AddressPool` configuration is deprecated. 
+
+In 4.10 layer2 and bgp IP addresses using the `AddressPool` configuration were assigned from different address pools. In {product-version} layer2 and bgp IP addresses can be assigned from the same address pool.
```

Comment:
Should "layer2" be "layer 2"? Or, if it is referring to a code element, should it be in back ticks? This applies to both sentences in this line.